pages/admin/config.yml

```diff
@@ -0,0 +1,307 @@
+backend:
```

Comment:
What's this config.yml file for? Should it be documented in the README?

Reply:
This is for modifying collection items via the [Netlify admin pages](https://federalist.18f.gov/documentation/getting-started-with-netlify-cms/). Happy to add to the README if you think that'd clarify matters?

Reply:
Ah, I've never used it before. Yes, adding this to the README would be great.

Reply:
Added to README!